source/braille.py

```diff
@@ -1441,11 +1441,146 @@ def _setCursor(self, info: textInfos.TextInfo):
 class ReviewTextInfoRegion(TextInfoRegion):
 
 	allowPageTurns=False
+	_currentSelection: textInfos.TextInfo | None = None
+	_currentScriptName: str | None = None

```

Comment:
is there a reason why these are stored as script names and not just script objects? 
That way the sets could just be collections of scripts not script names.
It seems more pythonic to use the base object not one of its properties.